### PR TITLE
Bikeshed ssh

### PR DESCRIPTION
**Pull request by krzyzacy**
did it for all generated jobs + cadvisor presubmit (because copied the 1st job), try on them first while shedding on all other jobs - we need some other find/replace strategy, or might as well write a script + test

-3k lines to start with

/assign @BenTheElder

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -110,6 +110,24 @@ presets:
...
+  - name: USER
+    value: prow
+  - name: JENKINS_GCE_SSH_PRIVATE_KEY_FILE
+    value: /etc/ssh-key-secret/ssh-private
+  - name: JENKINS_GCE_SSH_PUBLIC_KEY_FILE
+    value: /etc/ssh-key-secret/ssh-public
+  volumes:
+  - name: ssh
+    secret:
+      defaultMode: 256
```
Comment (BenTheElder) [inline prow/config.yaml]: nit, can we use octal instead?

File: jobs/config.json [hunk 2]
```diff
@@ -9000,7 +9000,7 @@
       "--node-tests=true",
       "--provider=gce",
       "--test_args=--nodes=1 --focus=\"\\[Serial\\]\" --skip=\"\\[Flaky\\]|\\[Benchmark\\]\"",
-      "--timeout=300m"
+      "--timeout=180m"
```
Comment (BenTheElder) [inline jobs/config.json]: hmm is this intentional??

File: prow/config.yaml (hunk 1 above)
Comment (krzyzacy) [reply prow/config.yaml]: done

File: jobs/config.json (hunk 2 above)
Comment (krzyzacy) [reply jobs/config.json]: ref https://github.com/kubernetes/test-infra/pull/6046, cc @dashpole @yguo0905 this need to be in the generated config

Comment (yguo0905) [reply jobs/config.json]: I though the presubmit check would catch this in https://github.com/kubernetes/test-infra/pull/6046.

**Pull request merged**